From: Dmitry Shachnev <mitya57@debian.org>
Date: Sat, 8 Feb 2020 13:27:14 +0300
Subject: Remove setuptools-3.X entry point

It gets outdated with every new Python 3 default version, anyway.
---
 setup.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 765b7ae..29e29a8 100644
--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,6 @@ try:
         entry_points = {
         'console_scripts': [
             'nosetests = nose:run_exit',
-            'nosetests%s = nose:run_exit' % py_vers_tag,
             ],
         'distutils.commands': [
             ' nosetests = nose.commands:nosetests',
@@ -34,6 +33,9 @@ try:
         },
         test_suite = 'nose.collector',
         )
+    if sys.version_info.major == 2:
+        addl_args['entry_points']['console_scripts'].append(
+            'nosetests%s = nose:run_exit' % py_vers_tag)
     addl_args.update(extra)
 
     # This is required by multiprocess plugin; on Windows, if
